packages/recipes/README.md

```diff
@@ -0,0 +1,57 @@
+# `@fuel-ts/recipes`
```

Comment:
Do you see an external use case for this package? I'm trying to envision why a consumer would install this package in isolation.